Makefile

```diff
@@ -217,6 +217,12 @@ istio-integration-test: install deploy
 	# Increased TEST_REQUEST_TIMEOUT to 300s to avoid timeouts on newly created k3s clusters
 	cd tests/integration && TEST_REQUEST_TIMEOUT=300s && EXPORT_RESULT=true go test -v -timeout 35m -run TestIstioMain
 
+.PHONY: grpc-performance-test
+grpc-performance-test: install deploy
```

Comment:
Should we use install deploy here? Since this will not work out of the box as the image in the config will not work on Gardener. Also the istio module would not be installed on the cluster, because Istios default resource is not created.

I'd remove this make target and just use the documentation and targets in `tests/performance-grpc`.